src/commands/flow/internal/saveLog.ts

```diff
@@ -0,0 +1,59 @@
+import SASjs from '@sasjs/adapter/node'
+import { createFile, fileExists, generateTimestamp } from '@sasjs/utils'
+import path from 'path'
+import { displayError } from '../../../utils/displayResult'
+import { parseLogLines } from '../../../utils/utils'
+import { fetchLogFileContent } from '../../shared/fetchLogFileContent'
+import { generateFileName } from './generateFileName'
+
+// REFACTOR: move to utility
+export const saveLog = async (
+  links: any[],
+  flowName: string,
+  jobLocation: string,
+  logFolder: string,
+  sasjs: SASjs,
+  serverUrl: string,
+  access_token: string,
+  lineCount: number = 1000000
+) => {
+  return new Promise(async (resolve, reject) => {
```

Comment:
Because this function is marked as `async` it will return a `Promise` anyway, so we don't need to return a `Promise` explicitly

Reply:
fixed